ui-library-click-to-call/README.md

```diff
@@ -47,3 +53,1123 @@ Instead, it will copy all the configuration files and the transitive dependencie
+          </Text>
+          <img
+            style={{ width: "7rem", height: "auto" }}
+            src={hero}
+            alt="kcup logo"
+          />
+        </Stack>
+
+        <Text>
+          Welcome to a Click to Call sample for the Azure Communications UI
+          Library. Sample has the ability to:
+        </Text>
+        <ul>
+          <li>
+            Adhoc call teams users with a tenant set that allows for external
+            calls
+          </li>
+          <li>Joining Teams interop meetings as a Azure Communications user</li>
+          <li>Make a click to call PSTN call to a help phone line</li>
+          <li>Join a Azure Communications group call</li>
+        </ul>
+        <Text>
+          As a user all you need to do is click the widget below, enter your
+          display name for the call - this will act as your caller id, and
+          action the <b>start call</b> button.
+        </Text>
+      </Stack>
+      <Stack
+        horizontal
+        tokens={{ childrenGap: "1.5rem" }}
+        style={{ overflow: "hidden", margin: "auto" }}
+      >
+        <ClickToCallComponent
+          onRenderLogo={() => {
+            return (
+              <img
+                style={{ height: "4rem", width: "4rem", margin: "auto" }}
+                src={hero}
+                alt="logo"
+              />
+            );
+          }}
+          onSetDisplayName={setUserDisplayName}
+          onSetUseVideo={setUseVideo}
+        />
+      </Stack>
+    </Stack>
+  );
+};
+```
+
+Once we have made this we will need to add the new view to the root of the app `App.tsx`:
+
+`App.tsx`
+
+```typescript
+    ...
+    switch (page) {
+        case 'click-to-call': {
+           if (!token || !userId || !locator || startSession === undefined) {
+            return (
+            <Stack style={{height: '100%', width: '100%'}}>
+                <Spinner label={'Getting user credentials from server'} ariaLive="assertive" labelPosition="top" />;
+            </Stack>
+            )
+        }
+        return <ClickToCallScreen token={token} userId={userId} callLocator={locator} alternateCallerId={alternateCallerId}/>;
+    }
+    ...
+```
+
+Then if you have done this you should see this when the app is running:
+
+<img src='./tutorialImages/Sample-app-splash.png' width='800'>
+
+Then when you action the widget button you should see:
+
+<img src='./tutorialImages/Sample-app-widget-open.png' width='800'>
+
+Next we will talk about what we need to add to make this experience start a call in a new window.
+
+## Part 2 Creating a New Window Experience
+
+Now that we have a running application with our widget on the home page we will talk about starting the calling experience for your users with a new window. This scenario allows you to give your customer the ability to browse while still seeing your call in a new window. This can be useful in situations similar to when your users will use video and screensharing.
```

Comment:
nit: Now that we have a running application with our widget on the home page, we will ...